Update to msix terminology (#64992)
Azure/azure-content: articles/virtual-desktop/app-attach-glossary.md

An MSIX share is a network share that holds expanded MSIX packages. MSIX shares support SMB 3 or later. Applications get staged from this MSIX share without having to move application files to the system drive.

## MSIX image

> An MSIX image is a VHD/VHDx or a CIM file that contains one or more MSIX packaged applications. Each applications is delivered in the MSIX image via the MSIXMGR tool.

Reviewer: ```suggestion
An MSIX image is a VHD, VHDx, or CIM file that contains one or more MSIX packaged applications. Each application is delivered in the MSIX image using the MSIXMGR tool.
```

## Repackage

Repackaging takes a non-MSIX application and converts it into MSIX using the MSIX Packaging Tool (MPT). For more information, see [MSIX Packaging Tool overview](/windows/msix/packaging-tool/tool-overview).

> ## Expanding a MSIX package

Reviewer: ```suggestion
## Expand an MSIX package
```

> Expanding MSIX package is a multi-step process. It takes the MSIX file and put its content into a VHD(x) or CIM file. 

Reviewer: ```suggestion
Expanding an MSIX package is a multi-step process. Expansion takes the MSIX file and puts its content into a VHD(x) or CIM file. 
```
